kevm-pyk/src/kontrol/foundry.py

```diff
@@ -111,6 +111,9 @@ def contracts(self) -> dict[str, Contract]:
             contract_name = json_path.split('/')[-1]
             contract_json = json.loads(Path(json_path).read_text())
             contract_name = contract_name[0:-5] if contract_name.endswith('.json') else contract_name
+            if _contracts.get(contract_name) is not None:
+                raise RuntimeError('Project has some duplicated contract names that may clash with K definitions.')
```

Comment:
```suggestion
                raise RuntimeError('Project contains duplicated contract names that may clash in K definitions.')
```
What do you think of the slightly paraphrased formulation? The original version works too, but I read it as "it might clash with K keywords", which you've already fixed. Feel free to ignore this suggestion :).

Reply:
That works well thank you!